src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -66,16 +80,45 @@ private static void readPropertiesFromFile(String[] args) {
 		String propFileLocation;
 		if(args.length == 0) {
 			M_log.error("Command line arguments are not provided");
-			System.exit(0);
+			System.exit(1);
 		}
 		propFileLocation = args[0];
 		properties = Utils.getPropertiesFromURL(propFileLocation);
 		canvasToken = properties.getProperty(CANVAS_TOKEN);
 		canvasURL = properties.getProperty(CANVAS_URL);
 		esbURL=properties.getProperty(ESB_URL);
-		sisTermIdFromPropsFile = properties.getProperty(SIS_TERM_ID);
+		getTermsInfoFromProperties(properties);
 		checkForApiType(properties);
 	}
+	/*
+	 * The Terms that needs for course deletion are listed in the properties file. Each term has 2 piece of information 
+	 * term1=2030;04/06/15, the sisTermCode and the endDate of that term. We will check if the current system date is
+	 * past the endDate then that term becomes eligible for deleting the unused/unpublished courses that are created 
+	 * as apart of auto-provisioning of courses in canvas
+	 * 
+	 */
+	private static void getTermsInfoFromProperties(Properties properties) {
+		M_log.debug("getTermsInfoFromProperties(): called");
+		String termCount = properties.getProperty(TERM_COUNT);
+		if(termCount.isEmpty()) {
+			M_log.error("The property \"term.count\" is not set in the properties file \"canvasCourseDelete.properties\"");
+			System.exit(1);
+		}
+		int termCountInt = Integer.parseInt(termCount);
+		for(int i=1;i<=termCountInt;i++) {
+			String term = TERM+String.valueOf(i);
```

Comment:
Maybe this should be more pedantic, like "term_property_name".  The value isn't actually a term.